content/docs/components/tag/theming.mdx

```diff
@@ -3,4 +3,145 @@ id: tag
 scope: theming
 ---
 
-TODO
+## Overview
+
+The `Tag` component is a single part component. All of the styling is
```

Comment:
The `Tag` component is actually a multipart component. That's why your sandbox doesn't work. It has the following parts:
```markdown
- `container`
- `label`
- `closeButton`
```

Check out the Tag theme to figure out how you can style it: https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/tag.ts